### ISSUE TITLE
About the F30K dataset

### ISSUE THREAD

**Issue by songbohui** [kuanghuei/SCAN#63]
![Flickr 30K](https://github.com/kuanghuei/SCAN/assets/82745821/c4d102e5-2bfa-482a-a7aa-b1c5ce3047d0)
I would like to ask you, is this the target feature data set obtained from the F30K data through the bottom-up attention model?

**Comment by luna0804** [comment]
Yes. Image features extracted from faster r-cnn are included in the links provided by the authors
